tests/runtests/aux/config.sh

```diff
@@ -77,5 +77,6 @@ export PACKAGES="abrt \
                  libreport-plugin-logger \
                  libreport-plugin-mantisbt \
                  libreport-plugin-systemd-journal \
+                 python3-pyftpdlib \
```

Comment:
We don't really need the `pyftpdlib` to be available for all tests, just for one.

Can you instead install the package in `rlPhaseStartSetup`? And maybe remove it in `rlPhaseStartCleanup`.

Reply:
I added a conditional removal, since blindly removing packages is not nice.